### PR TITLE
Implement DEX pallet in the trappist runtime

### LINKED ISSUE #43: Implement DEX pallet in the trappist runtime.
Implement DEX pallet located in https://github.com/Wiezzel/substrate-dex into Trappist runtime.

In case there are some parameters to define let's bring the discussion here.

### PR DESCRIPTION
**Pull request by bidzyyys**
Closes #43

### CODE REVIEW

**Comment by arturgontijo** [comment]
@bidzyyys I see some format changes, I think we should keep running `cargo fmt` in the whole code base to keep a standard.

**Comment by bidzyyys** [comment]
> @bidzyyys I see some format changes, I think we should keep running `cargo fmt` in the whole code base to keep a standard.

@arturgontijo but I run `cargo fmt` before issuing this PR

**Comment by arturgontijo** [comment]
That's odd, maybe it's something on my end so:
```
➜  trappist git:(10da254) cargo fmt        
➜  trappist git:(10da254) ✗ git status       
HEAD detached at upstream/43-implement-dex-pallet-in-the-trappist-runtime
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   node/src/command.rs
	modified:   node/src/service.rs
	modified:   runtime/src/xcm_config.rs
```

Checking substrate's [node/cli/src/command.rs#L105-L150](https://github.com/paritytech/substrate/blob/master/bin/node/cli/src/command.rs#L105-L150) your way seems to be the correct one. Even with `BenchmarkCmd::Machine(cmd) =>` not having `{ ... }` there.

So, I think we can leave it as it is now and then take a further look at this `fmt` thing. =)

LGTM

**Review by arturgontijo — DISMISSED** [review]

File: runtime/Cargo.toml [hunk 1]
```diff
@@ -130,13 +133,14 @@ std = [
     "pallet-contracts/std",
     "pallet-contracts-primitives/std",
     "pallet-contracts-rpc-runtime-api/std",
-    "pallet-identity/std",
+    "pallet-dex/std",
+	"pallet-dex-rpc-runtime-api/std",
+	"pallet-identity/std",
 	"pallet-multisig/std",
```
[the remark this answers is not in the document]
Comment (bidzyyys) [reply runtime/Cargo.toml:139]: For me too, but I do not why; have a look -> https://github.com/paritytech/trappist/blob/43-implement-dex-pallet-in-the-trappist-runtime/runtime/Cargo.toml#L105:L166

**Comment by bidzyyys** [comment]
Blocked by https://github.com/Wiezzel/substrate-dex/pull/26

**Review by Wiezzel — APPROVED** [review]
LGTM

**Pull request merged**